README.md

```diff
@@ -1 +1,94 @@
-# Math-magicians
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math magicians App
+
+> A single page app consisting of a calculator.
+
+## Built With
+
+- Major languages: HTML CSS & JavaScript
+- Framework(Library): React
+- Technologies used
+
+## Getting Started
+
+- To get a local copy up and running follow these simple example steps.
+
+# Setup
+
+- git clone https://github.com/ppmarq1/Math-magicians.git
+
+# Usage
+
+- Usage: cd <https://github.com/ppmarq1/Math-magicians.git>, npm install, npm start.
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Have a working and updated browser
+- Have a local version control like **git** install on your computer
+- Have an account and sign in on GitHub, as online/remote version control system
+- Download waves to check project accessibilty
+
+### Setup
+
+-Clone the git repo using this command: **git clone github.com/ppmarq1/Math-magicians.git**
+
+### Install
+
+- Lighthouse (An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more).
+
+- Webhint (A customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors).
+
+- Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
+
+- ESlint (A mighty, modern linter that helps you avoid errors and enforce conventions in JavaScript codes)
+
+### Create React App
+
+- npx create-react-app my-app
+- cd my-app
+- npm start
+
+### Usage
+
+- Usage: cd <https://github.com/ppmarq1/Math-magicians.git>, npm install, npm start.
+
+### Run tests
+
+- **npx hint .** : To test html codes
+- **npx stylelint "**/\*.{css,scss}"\*\* : To test css code
+- **npx eslint .** : To test Eslint Report
+
+### Deployment
+
+- Go to settings
+- Click on pages
+- Choose your source
+- Choose folder
+- choose a theme(optional)
+- Enforce HTTPS
+- And then save
+
+## Author
+
+👤 **Pedro Marques**
+
+- GitHub: https://github.com/ppmarq1
+- Twitter: https://twitter.com/ppmarq1
+- LinkedIn: https://br.linkedin.com/in/pedroalmeidamarques/
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ppmarq1/Math-magicians/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] This `MIT link` is broken. I would recommend fixing this to make README professional.

Reply:
Resolved. Thank you!